components/uid-title/src/UidTitle.ts

```diff
@@ -0,0 +1,64 @@
+import {css, html, TemplateResult} from 'lit';
+import {property} from 'lit/decorators.js';
+import {unsafeHTML} from "lit-html/directives/unsafe-html.js";
+import {UidElement} from "@bonitasoft/uid-common/dist/src/common/UidElement";
+import {Alignment, Level} from "./UidTitleType";
+
+/**
+ * @element uid-title
+ * Input field, optionally with a label, where the user can enter information
+ */
+export class UidTitle extends UidElement {
+
+  private name = 'uidTitle';
+
+  @property({attribute: 'id', type: String, reflect: true})
+  id: string = '';
+
+  // User should take care to sanitize the 'text' content before using this.
+  // See e.g. https://github.com/google/closure-library/blob/master/closure/goog/html/sanitizer/htmlsanitizer.js
+  @property({attribute: 'allow-html', type: Boolean, reflect: true})
+  allowHTML: boolean = false;
+
+  @property({attribute: 'text', type: String, reflect: true})
+  text: string = 'Title';
+
+  @property({attribute: 'alignment', type: String, reflect: true})
+  alignment: Alignment = Alignment.LEFT;
+
+  @property({attribute: 'level', type: String, reflect: true})
+  level: Level =Level.LEVEL1;
+
+  static styles = [
+    UidElement.styles,
+    css``
+  ];
+
+  render(): TemplateResult {
+    switch (this.level) {
```

Comment:
I suggest to do a toLowerCase() on level, and perhaps remove the potential spaces?
It would handle typos smoothly 😃